client/webserver/site/src/js/order.ts

```diff
@@ -257,14 +259,21 @@ export default class OrderPage extends BasePage {
     tmpl.status.textContent = OrderUtil.matchStatusString(m)
 
     const setCoin = (pendingName: string, linkName: string, coin: Coin) => {
+      const formatCoinID = (cid: string) => {
+        if (cid.startsWith(coinIDTakerFoundMakerRedemption)) {
+          const makerAddr = cid.substring(coinIDTakerFoundMakerRedemption.length)
+          return `${makerAddr} redeemed`
```

Comment:
Should format via **intl**. Recommend format `redeemed by: [addr]`

Reply:
Now re-tested (also checked that similar flow isn't broken, if done in reverse for the same trading pair - that is when Taker finds Maker DCR redemption on its own):

<img width="300" alt="image" src="https://user-images.githubusercontent.com/112318969/208142088-0cae7825-55ad-49f2-93c9-cc163d114e83.png">